### DISCUSSION TITLE
Z-Wave LR frequency choice for Europa

### DISCUSSION THREAD

**Discussion by pquinqu** [zwave-js/zwave-js-ui#3721] [Q&A] [answered]
Hi,
Actually there's ni way to move a Zooz ZST39 Z-Wave USB stick to Long Range un Europa zone. There's no choice of type "Europe (long range)" in the z-wave js-ui settings nor in the settings of the stick.

The solution might be to add a "Europe (long range)" choice to the settings tab and to the local settings of the 800 sticks.
And to enable automatically that setting when a 800 LR stick (like ZST39) is detected.
And enable the correct protocol like for the USA (long range) one.

Thanks for your help.
Phil

**Comment by robertsLando** [comment, marked as answer]
This is expected, there isn't an official europe frequency for LR right now.

cc @AlCalzone

**Reply by AlCalzone** [reply]
It's coming though, probably end of this year or start of 2025. But then you also need to have LR capable end devices for Europe to make use of it.

**Reply by pquinqu** [reply]
ok, thanks for the replies.
I'm a bit confused as I recently bought a Zooz ZST39 to a french reseller.
The stick operates at 868,42Mhz which is the European Z-Wave RF.
The user manual writes : "Extended range up to a mile in open space".

So I backuped the NVM of the "old" ZST10 700, replaced it by the new ZST39, mapped /dev/AMA0 (not /dev/ttyUSB0) to /dev/zwave in the docker-compose file, restarted the container and finally restored the NVM backup and ... "tout roule ma poule" (working perfect, eh eh thank you all devs for the hard work).

But no long range indicator in the protocol column, no LR in the mesh map, ...
Searching a bit around, fount that a LR choice exists for USA in the frequency settings but not Europa. And thought it was just a kind of oversight in the settings.

So, the only thing to do is to wait...
